src/bindings/python/tests/test_direct_comparison/test_partial_shape.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+from openvino import PartialShape
+
+def test_partial_shape_eq_list():
+    a = [1, 2, 3]
```

Comment:
Please, don't use one-later variables -- neither linters nor developers don't like it :)
Moreover these to tests may be merged. Please check [this doc](https://docs.pytest.org/en/7.1.x/example/parametrize.html)

Reply:
Done, Thank you!